### ISSUE TITLE
tar: Exiting with failure status due to previous errors

### ISSUE THREAD

**Issue by wzhen1996** [tensorflow/models#2100]
I have uncompressed the validation set of imagenet successfully, but when it comes to the training set it appears like the following:

Uncompressing individual train tar-balls in the training data.
Processing: n01440764
tar: Skipping to next header 
tar: Exiting with failure status due to previous errors

**Comment by shivaniag** [comment]
Could you please provide more details on this, e.g. code snippet and dataset link.

**Comment by wzhen1996** [comment]
I'm trying to download the imagenet dataset following the instructions in inception directory.

**Comment by rohan100jain** [comment]
Could you provide the exact commands you issued so that we can try and reproduce?

**Comment by davidparks21** [comment]
I've downloaded imagenet twice now and am getting the same thing, download from image-net.org via:

`wget -c http://www.image-net.org/archive/stanford/fall11_whole.tar`

**Comment by ymodak** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by davidparks21** [comment]
A followup note on this issue, after contacting the team at stanford they've added an MD5 hash of the download so you can now validate the download. I had numerous corrupt downloads, but finally got a good download that I was able to vaidate against their MD5 hash successfully.
